### DISCUSSION TITLE
Keeping widgets on a workspace in sway

### DISCUSSION THREAD

**Discussion by ErikBolumburu** [elkowar/eww#208] [Q&A]
Is there a way to keep my widgets out of the way on workspace one when using the sway, wayland window manager?

**Comment by jkim5959** [comment]
I hope you found an answer. I want to know how to do this too.

**Comment by IceAsteroid** [comment]
To keep eww always spawned in the same workspace when every time sway starts, write a script like follows, and specify it in sway conf.

The script below is to utilize the ability of sway to commit actions in command line by commands.

```
# In sway config 
exec "sleep 0.2; <the_path_of_the_script>" 
```

```
swaymsg "workspace N" # N is which workspace that you want eww to be spawned.
    swaymsg "exec eww"
sleep 0.2 # prevent misplacing of the next application to spawn.
swaymsg "exec <second_app>"
sleep 0.2

swaymsg "workspace Z"
    swaymsg "exec <another_app>"
sleep 0.2

#... and so on
```

I haven't used eww yet, so the command to spawn eww might be incorrect, you should replace with the correct command.
